source/addComment.js

```diff
@@ -1,3 +1,39 @@
 $(document).ready(function(){
-  // add your code here
-})
+
+  $(document.body).append(
+   $('<form/>').prop('id', 'new_comment').append(
+     $("<textarea/>", {
+       type: 'text',
+       id: 'textArea',
+       placeholder: 'Your comment here...'
+       }),
+     $("<br/>"),
+     $("<input/>", {
+       type: 'text',
+       id: 'authorName',
+       placeholder: 'Author Name'
+       }),
+       $("<input/>", {
+         type: 'submit',
+         id: 'submit',
+         value: 'Submit'
+       })
+   )
+  )
+
+   $('#new_comment').submit(function() {
+    var comment = $('#textArea').val()
```

Comment:
rather than adding id's to the input element's themselves, you could select them by their input and the name associated with it (you'll probably come across this more often)